docs/UserGuide.md

```diff
@@ -408,46 +409,86 @@ DELIVERY_DATE)!_
 
 **Command fails (invalid_date_format):** _Please provide the date in the format: yyyy-MM-dd._
 
-### View all deliveries
+### View a list of deliveries
 
-Shows a list of all deliveries.
+> Lists the deliveries in the delivery book.
 
-**Format:** `delivery list [--status STATUS] [--customer CUSTOMER_ID] [--date EXPECTED_DELIVERY_DATE]  [--sort SORT]`
+If you would like to get an overview of all your deliveries, you can use `delivery list`{.swift} to list all your
+deliveries at once. You can also filter the list of deliveries by status, customer ID and expected delivery date.
+You can also sort the list of deliveries by expected delivery date in ascending or descending order. By default, the
+list of deliveries will be sorted by expected delivery date in descending order (latest first).
 
-**Example:** `delivery list --status created --customer 1 --date 2023-12-12 --sort desc`
+**Format:**
+`delivery list [--status STATUS] [--customer CUSTOMER_ID] [--date EXPECTED_DELIVERY_DATE]  [--sort SORT]`{.swift}
 
-**Accepted Values:**
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
 
-_STATUS_: CREATED/SHIPPED/COMPLETED/CANCELLED. If unspecified, defaults to show all deliveries.
+**Note:**
 
-_CUSTOMER_ID_: Integer
+* All fields are optional.
+* The delivery `STATUS`{.swift} accepts the following values: `CREATED`{.swift}/`SHIPPED`{.swift}/`COMPLETED`{.swift}/
+  `CANCELLED`{.swift}.
+* The `CUSTOMER_ID`{.swift} must be an integer.
+* The `EXPECTED_DELIVERY_DATE`{.swift} must be in yyyy-MM-dd format or `TODAY`{.swift} for today’s date.
+* The `SORT`{.swift} accepts the following values: `ASC`{.swift}/`DESC`{.swift}.
+  </box>
 
-_EXPECTED_DELIVERY_DATE_: Expected Delivery Date String in YYYY-MM-DD format or `today` for today’s date
+**Examples:**
 
-_SORT_: String of either `asc` for ascending or `desc` for descending or defaults to sort by expected delivery date.
+* `delivery list --status CREATED --customer 1 --date 2023-12-12 --sort DESC`{.swift} </br>
+  Lists all deliveries with status `CREATED`{.swift} for customer with ID `1`{.swift} and expected delivery date of 12
+  Dec 2023 in descending order.
 
-**Command succeeds (>0 deliveries):**
-![](images/delivery/delivery_list.png)
+* `delivery list --status SHIPPED --date TODAY`{.swift} </br>
+  Lists all deliveries with status `SHIPPED`{.swift} for all customers and expected delivery date of today in ascending
+  order.
```

Comment:
If it's all of the same date, its sorted by ascending order of which characteristic? Is it delivery name?

Reply:
Actually, it doesn't make sense to specify sorting here as it has been filtered by 1 date already.